testutils/ptest2/src/main/resources/log4j2.properties

```diff
@@ -51,7 +51,7 @@ appender.FILE.strategy.max = 1
 loggers = Http, SpringFramework, OrgJclouds, Jclouds, Hive, NIOServerCnxn, ClientCnxnSocketNIO
 
 logger.Http.name = org.apache.http
-logger.Http.level = TRACE
+logger.Http.level = INFO
```

Comment:
I don't don't think we need this anymore, ptest2 belongs to the old ptest framework